bitnami/spring-cloud-dataflow/README.md

```diff
@@ -395,7 +395,7 @@ externalDatabase.dataflow.database=myskipperdatabase
 
 NOTE: When using the indidual propertes (scheme, host, port, database, an optional jdbcParameters) this chart will format the JDBC URL as `jdbc:{scheme}://{host}:{port}/{database}{jdbcParameters}`. The URL format follows that of the MariaDB database drive but may not work for other database vendors.
 
-To use an alternate database vendor (other than MariaDB) you can use the `externalDatabase.dataflow.url` and `externalDatabase.skipper.url` properties to provide the JDBC URLs for the dataflow server and skipper respectively. If these properties are defined, they will take precendence over the individual attributes. As an example of configuring an external MS SQL Server database:
```

Comment:
Shouldn't this be "precedence" instead of "formatting"?

Reply:
I fixed this typo in order to get this landed, let me know if you think the previous value was actually correct.

Reply:
@marcosbc Yes, you're right! Funny that it was "formatting" 😄 

I think that was the original author's mistake because codespell wouldn't correct a typo version of "precedence" to "formatting".

But you really reviewed this carefully.  I wouldn't have caught that.  Great Job! 👍👍👍